ls1intum/Artemis
# `Programming exercises`: Fix an issue when creating external submissions

Prevent a LazyInitializationException by using the correct exercise object

Fixes #4411 

TODO: we should create a server integration test that fails with the current version on `develop` and passes with the changes in this PR

### Checklist
#### General



### Steps for Testing

Prerequisites:
1. Create a programming exercise
2. Add an external submission
3. Make sure everything works correctly (check the participation, submission and scores page)

### Review Progress





#### Code Review
#### Manual Tests
